### PR TITLE
439-fix: Correct principle card behavior

### LINKED ISSUE #439: Principle cards expand only on hover
## 🐛 Describe the Bug

On main page cards explaining principles of the school expand only on hover, so no way to read their content on mobile devices.

## 🔄 Steps to Reproduce

Provide the steps to reproduce the behavior:
1. Navigate to '[main page](https://pr433.rs.school/#about)'
2. Hover on 'Open to everyone' card.
3. Repeat on mobile or set 'mobile' in devtools
4. Observe the error

## 🏁 Expected Behavior

Show all content in card if device doesnt support hover.

## 📸 Screenshots

image

### PR DESCRIPTION
**Pull request by thirdmadman**
## What type of PR is this? (select all that apply)

- [ ] 🍕 Feature
- [x] 🐛 Bug Fix
- [ ] 🚧 Breaking Change
- [ ] 🧑‍💻 Code Refactor
- [ ] 📝 Documentation Update

## Description

Corrected PrincipleCard and Principles according to new layout from figma.

## Related Tickets & Documents

- Related Issue #439
- Closes #439

### Desktop
image
image

### Laptop
image
image

### Mobile
image
image

## Added/updated tests?

- [ ] 👌 Yes
- [x] 🙅‍♂️ No, because they aren't needed
- [ ] 🙋‍♂️ No, because I need help

## Summary by CodeRabbit

- **New Features**
	- Enhanced the visual hierarchy and responsiveness of the principle card component.
	- Introduced a new card header structure for better organization of content.
	- Improved spacing and layout consistency across different screen sizes.

- **Bug Fixes**
	- Simplified hover interactions for a more straightforward user experience.

- **Style**
	- Updated styles for the principle card, including adjustments to padding, alignment, and layout properties.

### CODE REVIEW

**Review by Wystov — APPROVED** [review]

**Review by SpaNb4 — APPROVED** [review]

**Review by ansivgit — APPROVED** [review]
Approve after fix sizes

File: src/widgets/principles/ui/principle-card/principle-card.scss [hunk 1]
```diff
@@ -9,29 +9,43 @@
...
   flex-flow: column nowrap;
   place-content: center flex-start;
-  align-items: center;
+  align-items: flex-start;
 
-  width: 380px;
-  height: 240px;
-  padding: 36px 24px 24px;
+  width: 100%;
+  padding: 23px;
```
[the remark this answers is not in the document]
Comment (thirdmadman) [reply src/widgets/principles/ui/principle-card/principle-card.scss]: fixed in https://github.com/rolling-scopes/site/pull/483/commits/de6682258bea36b6bc1f39b6ffff27046c1c4576

File: src/widgets/principles/ui/principle-card/principle-card.scss [hunk 2]
```diff
@@ -48,6 +48,11 @@
     letter-spacing: 0;
   }
 
+  & span img {
```
Comment (ansivgit) [inline src/widgets/principles/ui/principle-card/principle-card.scss:51]:
```suggestion
  span img {
```

Comment (thirdmadman) [reply src/widgets/principles/ui/principle-card/principle-card.scss]: I used same logic as in other code. "Ugly but consistent"

fixed in: https://github.com/rolling-scopes/site/pull/483/commits/ad2cbb2fc7168ba88022942afe8f290019b89ab8

**Review by Yelunina — APPROVED** [review]

**Pull request merged**